pr-data.csv

```diff
@@ -1504,6 +1504,77 @@ https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-t
+https://github.com/apache/gora,6c6b06c916b969352a6757ba7025cc78055fa87a,gora-ignite,org.apache.gora.ignite.store.TestIgniteStore.testGetWithFields,ID,,,
+https://github.com/apache/gora,6c6b06c916b969352a6757ba7025cc78055fa87a,gora-ignite,org.apache.gora.ignite.store.TestIgniteStore.testObjectFieldValue,ID,,,
+https://github.com/apache/gora,6c6b06c916b969352a6757ba7025cc78055fa87a,gora-ignite,org.apache.gora.ignite.store.TestIgniteStore.testPutArray,ID,,,
+https://github.com/apache/gora,6c6b06c916b969352a6757ba7025cc78055fa87a,gora-ignite,org.apache.gora.ignite.store.TestIgniteStore.testPutBytes,ID,,,
+https://github.com/apache/gora,6c6b06c916b969352a6757ba7025cc78055fa87a,gora-ignite,org.apache.gora.ignite.store.TestIgniteStore.testPutMap,ID,,,
+https://github.com/apache/gora,6c6b06c916b969352a6757ba7025cc78055fa87a,gora-ignite,org.apache.gora.ignite.store.TestIgniteStore.testPutMixedMaps,ID,,,
+https://github.com/apache/gora,6c6b06c916b969352a6757ba7025cc78055fa87a,gora-ignite,org.apache.gora.ignite.store.TestIgniteStore.testPutNested,ID,,,
+https://github.com/apache/gora,6c6b06c916b969352a6757ba7025cc78055fa87a,gora-ignite,org.apache.gora.ignite.store.TestIgniteStore.testQuery,ID,,,
+https://github.com/apache/gora,6c6b06c916b969352a6757ba7025cc78055fa87a,gora-ignite,org.apache.gora.ignite.store.TestIgniteStore.testQueryEndKey,ID,,,
+https://github.com/apache/gora,6c6b06c916b969352a6757ba7025cc78055fa87a,gora-ignite,org.apache.gora.ignite.store.TestIgniteStore.testQueryKeyRange,ID,,,
+https://github.com/apache/gora,6c6b06c916b969352a6757ba7025cc78055fa87a,gora-ignite,org.apache.gora.ignite.store.TestIgniteStore.testQueryStartKey,ID,,,
+https://github.com/apache/gora,6c6b06c916b969352a6757ba7025cc78055fa87a,gora-ignite,org.apache.gora.ignite.store.TestIgniteStore.testQueryWebPageSingleKey,ID,,,
+https://github.com/apache/gora,6c6b06c916b969352a6757ba7025cc78055fa87a,gora-ignite,org.apache.gora.ignite.store.TestIgniteStore.testQueryWebPageSingleKeyDefaultFields,ID,,,
+https://github.com/apache/gora,6c6b06c916b969352a6757ba7025cc78055fa87a,gora-ignite,org.apache.gora.ignite.store.TestIgniteStore.testResultSizeStartKey,ID,,,
+https://github.com/apache/gora,6c6b06c916b969352a6757ba7025cc78055fa87a,gora-ignite,org.apache.gora.ignite.store.TestIgniteStore.testResultSizeStartKeyWithLimit,ID,,,
+https://github.com/apache/gora,6c6b06c916b969352a6757ba7025cc78055fa87a,gora-ignite,org.apache.gora.ignite.store.TestIgniteStore.testUpdate,ID,,,
+https://github.com/apache/gora,6c6b06c916b969352a6757ba7025cc78055fa87a,gora-mongodb,org.apache.gora.mongodb.filters.DefaultFactoryTest.testCreateFilter_list_2,ID,,,
+https://github.com/apache/gora,6c6b06c916b969352a6757ba7025cc78055fa87a,gora-neo4j,org.apache.gora.neo4j.store.Neo4jStoreTest.org.apache.gora.neo4j.store.Neo4jStoreTest,ID,,,
+https://github.com/apache/gora,6c6b06c916b969352a6757ba7025cc78055fa87a,gora-orientdb,org.apache.gora.orientdb.mapreduce.TestOrientDBStoreWordCount.testFlinkWordCountFlink,ID,,,
```

Comment:
Could you point me to the file with logs for this test? Thanks.

Reply:
`~/new-tests/gora/gora-ignite.nondex_log`
`~/new-tests/gora/gora-mongodb.nondex_log`
`~/new-tests/gora/gora-neo4j.nondex_log`
`~/new-tests/gora/gora-orientdb.nondex_log`

Reply:
I only found `testWordCount` but not `testFlinkWordCountFlink` in `~/new-tests/gora/gora-orientdb.nondex_log`

Reply:
Yes, the fewer tests are run the more likely that results are deterministic. Keep all logs.